main.go

```diff
@@ -97,9 +97,9 @@ func main() {
 	// namespace.
 	flag.StringVar(&watchNamespace, "namespace", os.Getenv("WATCH_NAMESPACE"),
 		"Namespace that the controller watches to reconcile host resources.")
-	flag.StringVar(&metricsAddr, "metrics-addr", "127.0.0.1:8085",
+	flag.StringVar(&metricsBindAddr, "metrics-bind-address", "127.0.0.1:8085",
 		"The address the metric endpoint binds to.")
-	flag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
+	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
```

Comment:
Changing these flags is going to break people. Notwithstanding that these are the new defaults, I don't think there's any actual problem with keeping the ones we had is there?

Reply:
Hi, these are auto generated and we are using the same names in CAPI and CAPM3 as well. However, I am ok with keeping the old flags or change it when we have introduced releases in this repo so that it doesn't break any existing workflow.